website/docs/quick-start.md

```diff
@@ -31,32 +31,37 @@ This sample illustrates how to patch containers using vulnerability reports with
     ```bash
     trivy image --vuln-type os --ignore-unfixed mcr.microsoft.com/oss/nginx/nginx:1.21.6
 
-3. Patch the image using the Trivy report. You will need to start `buildkitd` if it is not already running:
-
+3. Patch the image using the Trivy report.:
+
+    You'll need a buildkit instance to connect to.
+    By default copa will attempt to auto-connect to an instance in order:
+        1. Default docker buildkit endpoint
```

Comment:
this requires 24.0 and the containerd-snapshotter, is that right? should we add this here or create a seperate section?

Reply:
Added note here.